docker/pipelines_runner/pipelines_runner.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+set -euo pipefail
+
+# Copyright 2019 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#################################################
+# Parses arguments.
+# Arguments:
+#   It is expected that this is called with $@ of the main script.
+#################################################
+function parse_args {
+  while [[ "$#" -gt 0 ]]; do
+    case "$1" in
+      --project)
+        google_cloud_project="$2"
+        ;;
+
+      --temp_location)
+        temp_location="$2"
+        ;;
+
+      --docker_image)
+        vt_docker_image="$2"
+        ;;
+
+      --zones)
+        zones="$2"
+        ;;
+
+      *)
+        command="$@"
+        break
+        ;;
+    esac
+    shift 2
+  done
+}
+
+function main {
+  options=`getopt -o '' -l project:,temp_location:,docker_image:,zones: -- "$@"`
+  parse_args "$@"
```

Comment:
shouldn't you pass $options not $@ here?

Reply:
We can. `getopt` inserts `--` between the long arguments and the command (i.e., 
`vcf_to_bq --input_pattern ...`), which I find hard to understand, so I used "$@" instead. Why do u prefer $options?

Reply:
Got it - as discussed offline, let's just add a comment that we are using `getopt` only for checking (and drop options=)

Reply:
Done.

Reply:
It would be best to put this inside parse_args since that function makes assumptions about the state of the arguments (ie, if the getopt line is removed, parse_args will fail).

Reply:
Done. Thanks!